eng/build.ps1

```diff
@@ -19,8 +19,8 @@ try {
 
     # directly invoke npm tests
     # if (($null -ne $args) -and ($args.Contains("-test") -or $args.Contains("-t"))) {
-    #     TestUsingNPM "$PSScriptRoot\..\Microsoft.DotNet.Try.Client"
-    #     TestUsingNPM "$PSScriptRoot\..\Microsoft.DotNet.Try.js"
+    #     TestUsingNPM "$PSScriptRoot\..\src\Microsoft.DotNet.Try.Client"
+    #     TestUsingNPM "$PSScriptRoot\..\src\Microsoft.DotNet.Try.js"
```

Comment:
Do we actually use these files in some way or is this vestigial?

Reply:
checking with @brettfo  as i think we moved this part to yaml tasks

Reply:
`npm test` (or similar) is invoked via YAML in the following directories:

- `Microsoft.DotNet.Interactive.Js/`
- `dotnet-interactive-vscode/`

Otherwise those comments are from the `/try` repo and can be deleted.

Reply:
removed them